meson.build

```diff
@@ -182,6 +182,7 @@ conf.set_quoted('SYSTEM_SYSVINIT_PATH',                       sysvinit_path)
 conf.set_quoted('SYSTEM_SYSVRCND_PATH',                       sysvrcnd_path)
 conf.set_quoted('RC_LOCAL_SCRIPT_PATH_START',                 get_option('rc-local'))
 conf.set_quoted('RC_LOCAL_SCRIPT_PATH_STOP',                  get_option('halt-local'))
+conf.set_quoted('OK_COLOR',                                   to_upper(get_option('ok-color')))
```

Comment:
Why not set "ANSI_OK_COLOR" directly here? This additional level of indirection doesn't seem to serve any purpose.

Reply:
`OK_COLOR` contains `GREEN`, `BLUE` or `ORANGE`

ANSI_OK_COLOR is defined as

    #define ANSI_OK_COLOR "ANSI_" OK_COLOR

which sets `ANSI_OK_COLOR` to `"\x1B[0;32m"` if `OK_COLOR` is `GREEN`.

The purpose is this conversion:

    "GREEN" to "\x1B[0;32m"
    "BLUE" to "\x1B[0;34m"
    "ORANGE" to "\x1B[1;33m"

Reply:
Quick test case / example in C:

```c
#include <stdio.h>

#define COLOR "GREEN"
#define ANSI_GREEN "\x1B[0;32m"
#define OK_COLOR "ANSI_" COLOR
#define ANSI_STOP "\033[0m"

int main() {
    printf("[" ANSI_GREEN "OK" ANSI_STOP "]\n");
    return 0;
}
```

Reply:
I agree, will fix.